integration/restart.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
I think these .sh scripts are in here twice now?

Reply:
yep! it's meant to be copied anywhere that needs it